src/Dedge.Cardidy/Model/Cards.cs

```diff
@@ -124,3 +124,13 @@ internal record UzCard : ACard
 {
     public UzCard() : base(CardType.UzCard, 8600, Sixteen) { }
 }
+
+internal record DinersClubInternational : ACard
+{
+    public DinersClubInternational() : base(CardType.DinersClubInternational, 36, From14To19) { }
+}
+
+internal record DinersClubUsAndCanada : ACard

```

Comment:
Both Diners card are validated with Luhn algorithm. For each, use `ALuhnCard` instead of `ACard`.

Reply:
Didn't catch that, good work